xml/core_registers.xml

```diff
@@ -22,7 +32,15 @@
             When 1, {\tt ebreak} instructions in User/Application Mode enter
             Debug Mode.
         </field>
-        <field name="0" bits="11" access="R" reset="0" />
+        <field name="intdisable" bits="11" access="R/W" reset="0">
+            0: Interrupts are handled as usual.
+
+            1: Interrupts are disabled. This bit supersedes mstatus.ie.
+
+            This feature may not exist in implementations from early in the
```

Comment:
The way I wrote the bit, it doesn't solely apply to single step. It seemed like it'd be simpler to implement that way.

I made it a disable so that existing implementations which have a 0 here will just be compliant.

We can revisit either of those decisions.